README.md

```diff
@@ -0,0 +1,40 @@
+![gusto logo](./assets/Gusto_logo.png)
+
+# Gusto Python Client
+**Payroll, HR, benefits. Simplified**
+
+> [!NOTE]
+> Exciting news! Our first Python library beta release is here.
+
+## Authentication
```

Comment:
Replace this with high level features of the api client, look at stripe for example https://github.com/stripe/stripe-python